app/components/candidate_interface/application_dashboard_course_choices_component.rb

```diff
@@ -105,6 +105,7 @@ def conditions_row(application_choice)
           OfferConditionsReviewComponent.new(
             conditions: application_choice.offer.conditions_text,
             provider: application_choice.current_course.provider.name,
+            application_form: application_choice.application_form,
```

Comment:
I've done this in order to get the feature flag method on the conditions component. Not a huge fan of this... it is temporary but interested to hear alternatives?

Reply:
Not a huge fan either, but unfortunately, I think is necessary to be there.